tutorials/exclusion_mask.ipynb

```diff
@@ -0,0 +1,348 @@
+    "\n",
+    "We can also build our list of regions from a list of catalog sources. Here we use the Fermi 4FGL catalog which we read using `~gammapy.catalog.SourceCatalog`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fgl = SOURCE_CATALOGS[\"4fgl\"]()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now select sources that are contained in the region we are interested in."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "inside_geom = geom.contains(fgl.positions)\n",
+    "idx = np.where(inside_geom)[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now create the list of regions using our 0.3 degree radius a priori value. If the sources were extended, one would have to adapt the sizes to account for the larger size."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "exclusion_radius = Angle(\"0.3 deg\")\n",
+    "regions = [CircleSkyRegion(fgl[i].position, exclusion_radius) for i in idx]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can build the mask map the same way as above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mask_data = geom.region_mask(regions, inside=False)\n",
+    "mask_map = Map.from_geom(geom, data=mask_data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mask_map.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create the mask from statistically significant pixels in a dataset\n",
+    "\n",
+    "Here we want to determine an exclusion from the data directly. We will estimate the significance of the data and exclude all pixels above a given threshold.\n",
+    "\n",
+    "Here we use a dataset taken from Fermi data used in the 3FHL catalog. The dataset is already in the form of a `Datasets` object. We read it from disk. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "datasets = Datasets.read(\n",
+    "    \"$GAMMAPY_DATA/fermi-3fhl-crab/Fermi-LAT-3FHL_datasets.yaml\",\n",
+    "    \"$GAMMAPY_DATA/fermi-3fhl-crab/Fermi-LAT-3FHL_models.yaml\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We want to compute the significance per pixels for the data integrated over energy. We reduce the dataset to a simple image. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "datasets[0].psf = None\n",
```

Comment:
Why do you remove the `psf`? To improve computation time? Then maybe this can be mentioned because it can get a bit distracting otherwise...

Reply:
At the moment the `MapDataset.to_image()` fails if the `psf` is a `PSFKernel`. I just added a test that do not perform the `PSFKernel.to_image` to avoid the error.

The `PSFKernel.to_image()` needs to be implemented and a special handling has to be performed in `MapDataset.to_image()`